airflow/models/dagcode.py

```diff
@@ -124,15 +124,14 @@ def bulk_sync_to_db(cls, filelocs: Iterable[str], session=None):
             session.add(orm_dag_code)
 
         for fileloc in existing_filelocs:
-            old_version = existing_orm_dag_codes_by_fileloc_hashes[
-                filelocs_to_hashes[fileloc]
-            ]
-            file_modified = datetime.fromtimestamp(
-                os.path.getmtime(correct_maybe_zipped(fileloc)), tz=timezone.utc)
+            current_version = existing_orm_dag_codes_by_fileloc_hashes[filelocs_to_hashes[fileloc]]
+            file_mod_time = datetime.fromtimestamp(
+                os.path.getmtime(correct_maybe_zipped(fileloc)), tz=timezone.utc
+            )
 
-            if (file_modified - timedelta(seconds=120)) > old_version.last_updated:
```

Comment:
You will have to remove `timedelta` from the import statement at the top of the file. Pylint was failing with "import not used"

Reply:
Whoops, fixed